watertap/property_models/NaCl_prop_pack.py

```diff
@@ -688,7 +692,7 @@ def rule_diffus_phase(b):  # diffusivity, eq 6 in Bartholomew
                 + b.params.diffus_param["0"]
             )
 
-        self.eq_diffus_phase = Constraint(rule=rule_diffus_phase)
+        self.eq_diffus_phase_comp = Constraint(["NaCl"], rule=rule_diffus_phase_comp)
```

Comment:
Same here.

The rule above would also ideally not have `"Liq"` and `"NaCl"` hard-coded.

Reply:
> It's a minor thing for sure, but now a lot of our constraints have the "wrong" index set, i.e., it does not match that of the property it is calculating.

I agree with your comments. It probably needs to change, but I wonder if that should be for a subsequent PR. The most important part is the names of the properties because that is how a user/developer will interact with the properties in there own modules/models.

Reply:
@bknueven I think it is OK since this is the NaCl property model.

Reply:
To be clear, I didn't mean here, but in the rule for this constraint.

Again it's being nit-picky. Pyomo passes in the indices to the rule, so it's weird not to use them.